options.lua

```diff
@@ -593,6 +593,47 @@ local function about_menu(button_idx)
     }
   end
 
+  local function show_system_info()
+    ret = {
+      function()
+        GAME.backgroundImage = themes[config.theme].images.bg_readme
+        reset_filters()
+        local renderer_name, renderer_version, graphics_card_vendor, graphics_card_name = love.graphics.getRendererInfo()
+        local sys_info = {}
+        sys_info[#sys_info + 1] = {name = "Operating System", value = love.system.getOS()} 
+        sys_info[#sys_info + 1] = {name = "Renderer", value = renderer_name.." "..renderer_version}
+        sys_info[#sys_info + 1] = {name = "Graphics Card", value = graphics_card_name}
+        sys_info[#sys_info + 1] = {name = "LOVE Version", value = Game.loveVersionString()} 
+        sys_info[#sys_info + 1] = {name = "Panel Attack Engine Version", value = VERSION} 
+        sys_info[#sys_info + 1] = {name = "Panel Attack Release Version", value = GAME_UPDATER_GAME_VERSION} 
+        sys_info[#sys_info + 1] = {name = "Save Data Directory Path", value = love.filesystem.getSaveDirectory()}  
+        sys_info[#sys_info + 1] = {name = "Characters [Enabled/Total]", value = #characters_ids_for_current_theme.."/"..#characters_ids} 
+        sys_info[#sys_info + 1] = {name = "Stages [Enabled/Total]", value = #stages_ids_for_current_theme.."/"..#stages_ids} 
+        sys_info[#sys_info + 1] = {name = "Total Panel Sets", value = #panels_ids} 
+        sys_info[#sys_info + 1] = {name = "Total Themes", value = #found_themes}
+        local info_string = ""
+        for index, info in ipairs(sys_info) do
+          info_string = info_string .. info.name .. ": " .. (info.value or "Unknown") .. "\n"
+        end
+        while true do
+          gprint(info_string, 0, 15)
+          wait()
+          local panels_ret = nil
+          variable_step(
+            function()
+              if menu_escape() or menu_enter() then
+                panels_ret = {about_menu, {4}}
```

Comment:
```suggestion
                panels_ret = {about_menu, {5}}
```
In order to return back to the right spot